packages/sitecore-jss-nextjs/src/edge/personalize-middleware.ts

```diff
@@ -125,14 +133,19 @@ export class PersonalizeMiddleware {
     const language = req.nextUrl.locale || req.nextUrl.defaultLocale || 'en';
     let browserId = this.getBrowserId(req);
 
+    // Response will be provided if other middleware is run before us (e.g. redirects)
+    let response = res || NextResponse.next();
+
+    if (this.config.disabled && this.config.disabled(req, response)){
```

Comment:
Let's move this after the "personalize middleware start:" debug log for consistency/context (e.g. you should always see "personalize middleware start:" for every request, then the skipped reason).

Reply:
Okay. I thought it would be better to not print personalize start if we have a hard off switch (as opposed to other abort conditions which occur during personalize execution).